src/handlers/Item.cpp

```diff
@@ -315,20 +315,14 @@ void Item_Use(Character *character, PacketReader &reader)
 
 				if (level_up)
 				{
-					PacketBuilder builder(PACKET_RECOVER, PACKET_REPLY, 9);
```

Comment:
This declaration of `builder` was superseded by the declaration originally on line 329, so I think it can be safely deleted.

Reply:
I agree, looks like dead code.